ompi/group/group_init.c

```diff
@@ -209,20 +206,20 @@ ompi_group_t *ompi_group_allocate_bmap(int orig_group_size , int group_size)
         goto error_exit;
     }
     /* allocate the unsigned char list */
-    new_group->sparse_data.grp_bitmap.grp_bitmap_array = (unsigned char *)malloc
-        (sizeof(unsigned char) * ompi_group_div_ceil(orig_group_size,BSIZE));
+    new_group->sparse_data.grp_bitmap.grp_bitmap_array = (unsigned char *) malloc(
+        sizeof(unsigned char) * ompi_group_div_ceil(orig_group_size, BSIZE));
 
-    new_group->sparse_data.grp_bitmap.grp_bitmap_array_len =
-        ompi_group_div_ceil(orig_group_size,BSIZE);
+    new_group->sparse_data.grp_bitmap.grp_bitmap_array_len = ompi_group_div_ceil(orig_group_size,
+                                                                                 BSIZE);
```

Comment:
is there  a way to get clang-format to not change these type of assignment operations?